core/src/test/java/org/elasticsearch/action/bulk/BulkWithUpdatesIT.java

```diff
@@ -239,7 +239,6 @@ public void testBulkVersioning() throws Exception {
                         .setDoc("field", "3").setVersion(13).setVersionType(VersionType.INTERNAL))
                 .get();
 
-        assertThat(bulkResponse.getItems()[0].getFailureMessage(), containsString("version conflict"));
```

Comment:
this doesn't look like a rollback of https://github.com/elastic/elasticsearch/commit/b4cc3cd35dea2e3059142c6bd7e2eb13ca8944ac , are we sure it's ok?